Team2 project management with GitHub

hinckleyc opened the pull request:
Added information to:
 Assigning team members to issues in the GitHub issue tracker,
 Merging a branch or a fork to another branch or fork,
 Creating and reviewing a pull request,
 Conduct.
Please let me know if there is anything that needs to be fixed.

koscinskic committed:
Added commit message guidelines in assessment.md

All quality levels as well as an E-grade example for using appropriate commit messages were added

koscinskic committed:
Corrected indentation for commit message guidelines section in assessment.md

Commit:
Add guidelines to the 'creating and using a repository branch' section

Commit:
Finished the grading for merging branchs
or forks to another branch or fork

Commit:
add project management assigning issue tracker

Commit:
Merge branch 'team2-project-management-with-github' of github.com:Allegheny-Computer-Science-203-S2020/cs203-S2020-assessment into team2-project-management-with-github

TylerPham2000 committed:
Added conduct to group and creating and reviewing a pull request

TylerPham2000 committed:
Resubmitting the descriptions

Commit:
finished conflicts in the group section in assessment.md

Commit:
add emojis to communication and issue tracker

quit

Commit:
Merge branch 'team2-project-management-with-github' of github.com:Allegheny-Computer-Science-203-S2020/cs203-S2020-assessment into team2-project-management-with-github

Commit:
fixing line length

Commit:
fixing line length for build to pass

Commit:
Fixing travis ci errors in project management

Commit:
Fixing travis ci errors on line 50

Commit:
Fixed more travis ci errors

enpuyou reviewed (changes requested):
Please conform to the style guide here: https://github.com/Allegheny-Computer-Science-203-S2020/cs203-S2020-markdown

corlettim commented on assessment.md:
```diff
@@ -36,11 +36,34 @@ Project Management is Covered by Team 2
+  * G = when merging a branch or a fork to another branch or fork works most of
+  the time
+  * E = when merging a branch or a fork to another branch or fork works all the
+  time
```
I recommend rewording these. I also suggest making a standard format with capitalizing after the equal sign or not

hodevin replied:
It should be capitalized in this case.

koscinskic committed:
Removed Conduct guidelines from assessment.md

These guidelines are a component of the "Group" skills section, which is located on its own appropriately named branch. These guidelines are on said branch as well, so this removal is purely for branch clarity.

koscinskic committed:
Added lost guidelines for commit messages and repository branch usage in assessment.md

Restored the work of several lost commits be manually adding the content.

Added blank lines for .md file readability

koscinskic committed:
Added emoji rating symbols

Converted the original NIAGE ratings to the new emoji ratings.

Corrected line length to accomadate this change.

koscinskic commented:
I have resolved the requested changes concerning emoji style of assessment.

koscinskic committed:
Changed E symbol to :sunglasses: in guideline examples

Commit:
Finished writing for creating and using a repository branch

Commit:
Finished using the GitHub Flow model,Thomas did this I am just commiting

noorbuchi commented:
Hi, I'm a member of the markdown team and I've been trying to review some of the pull requests and make sure that the branches adhere to the markdown style in the markdown repository . There is a small issue with your pull request in which you are using bold in your writing. Please let me know if you any questions about markdown usage. Thanks!

Commit:
Finished using the Github issue tracker, thomas did this work

Commit:
fixed travis ci errors

Commit:
Finished Creating and assigning labels to
issues in the GitHub issue tracker, Anthony did this work

Commit:
Fixed travis ci errors

koscinskic committed:
Removed bold text in assessment.md

Changes made in adherence to Markdown guidelines

enpuyou committed:
Merge branch 'master' into team2-project-management-with-github

enpuyou committed:
Merge branch 'master' into team2-project-management-with-github

enpuyou commented on assessment.md:
```diff
@@ -32,15 +32,124 @@
+  other communication, issue descriptions occasionally confusing
+  * 😃 = use the issue tracker, issue descriptions not as in depth as they
+  should be
+  * 😎 = use the issue tracker, issue descriptions are in depth and clear
```
These two sections do not conform to the style guide. The first letter after equal sign should also be uppercase.

enpuyou commented on assessment.md:
```diff
@@ -32,15 +32,124 @@
 * Creating and assigning labels to issues in the GitHub issue tracker
+  * :disappointed: = Doing nothing.
+  * :confused:= Assigning but not clarifying tasks.
+  * :neutral_face: = Assigned tasks that are clear but are useless.
```
Why are both "assigning" and "assigned" both used here?

enpuyou commented on assessment.md:
```diff
@@ -32,15 +32,124 @@
+    understood.
+
 * Creating and assigning labels to issues in the GitHub issue tracker
+  * :disappointed: = Doing nothing.
```
I think it should be something more descriptive than just saying "doing nothing"

enpuyou commented on assessment.md:
```diff
@@ -32,15 +32,124 @@
+should be named according to their role (i.e.
+team2-project-management-with-github), and once that role is complete/no longer
+necessary, a pull request should be created to merge the branch with the main
+branch.
```
This section should be removed according to the markdown assessment template

koscinskic replied:
This specific issue (The presence of this section) is currently being addressed as an issue in the Markdown repository.

enpuyou commented on assessment.md:
```diff
@@ -32,15 +32,124 @@
+
 * Merging a branch or a fork to another branch or fork
+
+  * :disappointed: = Not making a branch when it your job
```
This sentence needs to be fixed

enpuyou commented on assessment.md:
```diff
@@ -32,15 +32,124 @@
+  * :smiley: = Mentions most related issues it is trying to close, describes
+    most changes and the purpose of the pull request
+  * :sunglasses: = Mentions all related issues it is trying to close, describes
+    all changes made and passes Travis CI
```
This section did not address "reviewing a pull request"

enpuyou commented on assessment.md:
```diff
@@ -32,15 +32,124 @@
+scheme and structure (e.g. action-content-file or equally descriptive format),
+consistently adhering to said structure, and adding commit descriptions if
+deemed necessary (e.g. Adding a list of feature additions / changes / removals
+or rationale for said actions).
```
This section should be removed

koscinskic replied:
This specific issue (The presence of this section) is currently being addressed as an issue in the Markdown repository.

enpuyou replied:
That's fine. I would suggest you fix the other issues first, and once the issue your team raised is being addressed, we can continue from there to work and merge this PR

enpuyou reviewed (changes requested):
This PR is not up to standard and many places below have to be fixed before merging. In general, there are inconsistent uses of the period at the end of each bullet point which should all be removed. All sections should also use present tense and third-person narrative

enpuyou commented:
Most commit messages in your group are also not up to standard in terms of the use of tense and uppercase, while the content was somewhat descriptive. Please review the standard for commit messages and write better commit messages in the future

koscinskic committed:
Removed Concrete Explanations for Technical Guidelines

*HIGHLY UNADVISED ACTION*
Done in compliance to Markdown Standards

ibarrezueta committed:
Merge branch 'master' into team2-project-management-with-github

gkapfham commented:
Hello @hinckleyc, this PR is not up-to-date with the master branch. Can you please do the update and then confirm that everything is still working correctly? Make sure that you finish this task today, okay?

stephensonc committed:
Merge branch 'master' into team2-project-management-with-github

Commit:
Fixed formatting

Commit:
Fixed travis errors and a PR error

enpuyou commented on assessment.md:
```diff
@@ -136,15 +136,113 @@
+  * :confused: = Created unclear repository forks.
+  * :neutral_face: = Created poorly named repository forks.
+  * :smiley: = Created useful named repository forks
+  * :sunglasses: = Created useful and descriptive repository forks
```
This part is still in the past tense

enpuyou reviewed (changes requested):
One more fix

koscinskic committed:
Corrected Tense for Repository Fork Guidelines in assessment.md

enpuyou committed:
Remove unneeded blank lines

enpuyou committed:
Remove unneeded line
